src/hooks/useLocalStorageReviews.js

```diff
@@ -0,0 +1,14 @@
+import { useState, useEffect } from 'react';
+
+export default function useLocalStorageReviews(key, initialValue) {
```

Comment:
This isn't really specific to reviews, we could pass any data in here. This is a helpful utility that should be named something generic, like `useStateWithLocalStoragePersistence` (or something shorter that captures the same idea), and we should rename `review` to `data` (also feel free to do something different here).